pyproject.toml

```
@@ -1,26 +1,48 @@
 [build-system]
-requires = ["hatchling", "hatch-requirements-txt"]
-build-backend = "hatchling.build"
+requires = ["poetry-core>=2.0.0,<3.0.0"]
+build-backend = "poetry.core.masonry.api"
 
 [project]
 name = "theoriq"
 version = "0.0.1"
 description = ""
 readme = "README.md"
-requires-python = ">=3.9"
+requires-python = ">=3.9,<4.0"
 authors = []
-dynamic = ["dependencies"]
 
-[tool.hatch.metadata]
-allow-direct-references = true
+dependencies = [
+    "biscuit-python @ git+https://github.com/chain-ml/biscuit-python.git@main",
```

Can we pin the `biscuit-python` version to a specific commit? This would help streamline future upgrades and make version management more predictable.
```suggestion
    "biscuit-python @ git+https://github.com/chain-ml/biscuit-python.git@2cd8292",
```